shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java

```diff
@@ -290,15 +287,15 @@
      * @return the int
      */
     @AliasFor(annotation = DubboService.class)
-    int retries() default DEFAULT_RETRIES;
+    int retries() default -1;
 
     /**
      * Alias for {@link DubboService#loadbalance()} .
      *
      * @return the string
      */
     @AliasFor(annotation = DubboService.class)
-    String loadbalance() default LoadbalanceRules.RANDOM;
+    String loadbalance() default LoadbalanceRules.EMPTY;
```

Comment:
why empty

Reply:
This is the default value for DubboService annotation. Empty and random are different. If the configuration file specifies load balancing is RoundRobin, then the final result for Empty is RoundRobin.